src/Lti/Lti11/Serializer/DeepLinking/LineItemSerializer.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Cerpus\EdlibResourceKit\Lti\Lti11\Serializer\DeepLinking;
+
+use Cerpus\EdlibResourceKit\Lti\Lti11\Context\DeepLinkingProps as Prop;
+use Cerpus\EdlibResourceKit\Lti\Message\DeepLinking\LineItem;
+
+final readonly class LineItemSerializer implements LineItemSerializerInterface
+{
+    public function __construct(
+        private ScoreConstraintsSerializer $scoreConstraintsSerializer = new ScoreConstraintsSerializer(),
```

Comment:
Constructor should accept any implementation of ScoreConstraintsSerializerInterface.